auth0/v3/authentication/database.py

```diff
@@ -60,25 +61,23 @@ def signup(self, client_id, email, password, connection, username=None,
             'email': email,
             'password': password,
             'connection': connection,
-            'username': username,
-            'user_metadata': user_metadata
         }
+        if username:
+            body.update({'username': username})
+        if user_metadata:
+            body.update({'user_metadata': user_metadata})
 
-        return self.post(
-            'https://{}/dbconnections/signup'.format(self.domain),
-            data=body
-        )
+        return self.post('https://{}/dbconnections/signup'.format(self.domain), data=body)
 
     def change_password(self, client_id, email, connection, password=None):
         """Asks to change a password for a given user.
         """
+        body = {
+            'client_id': client_id,
+            'email': email,
+            'connection': connection,
+        }
+        if password:
```

Comment:
I know you're just changing what's already here but `dbconnections/change_password` just sends a password change email, so I don't think it should take a password param